packages/frontend/src/hooks/useColumns.tsx

```diff
@@ -57,11 +57,11 @@ export const useColumns = (): TableColumn[] => {
     const resultsData = useExplorerContext(
         (context) => context.queryResults.data,
     );
-    const toggleSortField = useExplorerContext(
-        (context) => context.actions.toggleSortField,
+    const addSortField = useExplorerContext(
+        (context) => context.actions.addSortField,
     );
-    const setSortFields = useExplorerContext(
-        (context) => context.actions.setSortFields,
+    const removeSortField = useExplorerContext(
+        (context) => context.actions.removeSortField,
     );
```

Comment:
What if we moved this to the [ColumnHeaderContextMenu.tsx](https://github.com/lightdash/lightdash/pull/3125/files#diff-5233c6644e00e36d24b05be545314a4c9c1126d62bdf6eaf7e72e21a800002be) ? the table & header no longer needs to know about sorting so we could remove all the sorting properties from the header metadata.

Reply:
good call, I did not like that part as well. I just replaced existing setSortField and forgot about it